docs/framework/configure-apps/file-schema/runtime/loadfromremotesources-element.md

```diff
@@ -69,7 +69,7 @@ Specifies whether assemblies from remote sources should be granted full trust.
  The `<loadFromRemoteSources>` element lets you specify that the assemblies that would have run partially trusted in earlier versions of the .NET Framework are to be run fully trusted in the [!INCLUDE[net_v40_short](../../../../../includes/net-v40-short-md.md)] and later versions. By default, remote assemblies do not run in the [!INCLUDE[net_v40_short](../../../../../includes/net-v40-short-md.md)] and later. To run a remote assembly, you must either run it as fully trusted or create a sandboxed <xref:System.AppDomain> in which to run it.  
   
 > [!NOTE]
->  In the [!INCLUDE[net_v45](../../../../../includes/net-v45-md.md)], assemblies on local network shares are run as full trust by default; you do not have to enable the `<loadFromRemoteSources>` element.  
+>  In the [!INCLUDE[net_v45](../../../../../includes/net-v45-md.md)], assemblies on local network shares are run as full trust by default on workstation versions of Windows. Therefore, in many cases, you do not have to enable the `<loadFromRemoteSources>` element. However, it is still required for some versions of Windows Server or if the default .NET machine configuration has been customized.
```

Comment:
I do not think there were ever any differences between the workstation and server versions of Windows here.

Reply:
I agree this seems unlikely, but if our documentation is misleading people to think they don't need to care about a feature, there is some sort of improvement to be found.

Reply:
From @jeffschwMSFT discussion:

We should add the following:
- Explicitly call out that in .NET 4+ loadFromRemoteSources must be set to LoadFrom an assembly that comes a remote source.  The reason is that .NET 4+ no longer sandboxes the assembly in the way that 3.5 or earlier had, and to ensure that behavior was not relied on loadFromRemoteSources allows the load.
·       Add details of the exception message (there should only be one).